routes/index.js

```diff
@@ -29,11 +30,29 @@ routes.POSTregister = function(req, res, next) {
     if (err) {
       console.log('error while user register!', err); return next(err);
     }
-    console.log('user registered!');
-
-    passport.authenticate('local')(req, res, function () {
-      res.send({redirect: '/'});
+    
+    //send email verification
+    //Used example: https://github.com/heitortsergent/passport-local-mongoose-email/tree/master/examples/login
+    var authenticationURL = 'http://localhost:3000/verify?authToken=' + user.authToken;
```

Comment:
Should this be local host or the heroku page?  Or should it be able to switch between both?  (I'm actually not sure which is why I'm asking)